app/models/like.rb

```diff
@@ -0,0 +1,11 @@
+class Like < ApplicationRecord
+  belongs_to :user, class_name: 'User',
+                    foreign_key: 'author_id'
+  belongs_to :post, counter_cache: :likes_counter
+
```

Comment:
- [OPTIONAL] Kindly make use of one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - after_save to call the #update_likes_count method every time a new like has been saved to database, before you can make the update_likes_count method private